server/user.js

```diff
@@ -0,0 +1,85 @@
+const { TimeLogger } = require("../src/util");
+const { R } = require("redbean-node");
+const { UptimeKumaServer } = require("./uptime-kuma-server");
+const server = UptimeKumaServer.getInstance();
+const io = server.io;
+
+/**
+ * Send list of users to client
+ * @param {Socket} socket Socket.io socket instance
+ * @returns {Promise<Bean[]>}
+ */
+async function sendUserList(socket) {
+    const timeLogger = new TimeLogger();
+
+    let result = [];
+    let list = (await R.findAll("user")).map(({ id, username, active }) => ({
+        id,
+        username,
+        active
+    }));
+
+    io.to(socket.userID).emit("userList", result);
+
+    timeLogger.print("Send User List");
+
+    return list;
+}
+
+/**
+ * Send list of users to client
+ * @param {Socket} socket Socket.io socket instance
+ * @returns {Promise<Bean[]>}
+ */
+async function getUser(userID) {
+    const timeLogger = new TimeLogger();
+
+    const bean = await R.findOne("user", " id = ? ", [ userID ])
+        .then(({ id, username, active }) => ({
+            id,
+            username,
+            active
+        }));
+
+    if (!bean) {
+        throw new Error("User not found");
+    }
+
+    timeLogger.print(`Get user ${userID}`);
+
+    return bean;
+}
+
+/**
+ * Saves and updates given user entity
+ * @param {Socket} socket Socket.io socket instance
+ * @param user
+ * @returns {Promise<Bean[]>}
+ */
+async function saveUser(socket, user) {
+    const timeLogger = new TimeLogger();
+    const { id, username, active } = user;
+
+    const bean = await R.findOne("user", " id = ? ", [ id ]);
+
+    if (!bean) {
+        throw new Error("User not found");
+    }
+
+    username && (bean.username = username);
+    active !== undefined && (bean.active = active);
```

Comment:
What?
Could you please use if-statements to make this less obvuscated? ^^

Reply:
I'm personally very accustomed with this syntaxe which is perfectly valid in JS. I found it handy for a single instruction, where there is absolutely no doubt about what happens. In addition, linter rules of the project force me to be excessively verbose while using if-statements.

I can't simply write this:
`if (username) bean.username = username;`

I'm forced to write this:
```js
    if (username) {
        bean.username = username;
    }
```
Imagine if I have 15 variables to check?
So, I'm okay to change it by `if-statements`... but please relax this linter rule! 🥺

Reply:
Done.